[Term Entry] JavaScript Dates .getUTCDay() #2338 (#2456)
Codecademy/docs: content/javascript/concepts/dates/terms/getUTCDay/getUTCDay.md

---
Title: '.getUTCday()'
Description: 'Returns the day of the week in the specified date according to universal time.'
Subjects:
  - 'Web Development'
  - 'Computer Science'
Tags:
  - 'Date'
  - 'Methods'
CatalogContent:
  - 'introduction-to-javascript'

>   - 'paths/front-end-developer-career-path'

Reviewer: ```suggestion
  - 'paths/front-end-engineer-career-path'
```

---

> The ** `getUTCDay()` ** method returns the day of the week in the specified date according to universal time, as an integer value from 0 - 6 where 0 represents Sunday, 1 for Monday and so on.

Reviewer: ```suggestion
The **`getUTCDay()`** method returns the day of the week in the specified date according to universal time. The returned value is an integer from `0` to `6` where `0` represents Sunday, `1` for Monday and so on.
```

## Syntax

> ```javascript

Reviewer: ```suggestion
```pseudo
```

> getUTCday();

Reviewer: ```suggestion
date_obj.getUTCday();
```

> ```

Reviewer: add a line break between the block and the sentence

> The return value will give you a integer between 0 and 6:

Reviewer: ```suggestion
The return value of the passed `Date` object will yield an integer between `0` and `6`:
```

  * 0 = Sunday
  * 1 = Monday
  * 2 = Tuesday
  * 3 = Wednesday
  * 4 = Thursday
  * 5 = Friday
  * 6 = Saturday

## Example

The following example shows the method in use.

```javascript
const date = new Date('10 June 2023 14:15:30');
console.log(date.getUTCDay());
```

> The output will be 6.

Reviewer: ```suggestion
The output will be:

```shell
6
```
```
